src/main/java/discord4j/discordjson/json/UserData.java

```diff
@@ -20,8 +20,12 @@ static ImmutableUserData.Builder builder() {
 
     Id id();
 
+    @JsonProperty("global_name")
+    String globalName();
```

Comment:
even when the response is complete, display name may be null because the user has not set any display names